### DISCUSSION TITLE
More useful widgets ...maybe?

### DISCUSSION THREAD

**Discussion by ulliperk** [elkowar/eww#800] [Ideas]
Coming from Plasma 5 and use XFCE now ...
I am missing a rss widget and the "webslice" - can something like that be built with EWW ... and how?

**Comment by mantacid** [comment]
If you had a script that could update a file with RSS, you could use a listener variable to get the most recent entry, then do some black magic to transform the format into something eww can understand (I’d probably use a bunch of `sed` commands and some regular expressions, but thats likely far from the best way to do it).

addendum: There’s another discussion that asks a similar question, and from the responses it doesn’t look like we’ll be able to render html in any capacity anytime soon.
